packages/core/src/Stores/notification-store.js

```diff
@@ -55,6 +56,7 @@ export default class NotificationStore extends BaseStore {
                 root_store.common?.selected_contract_type,
                 root_store.client.is_eu,
                 root_store.client.has_enabled_two_fa,
+                this.p2p_order_props.order_id,
```

Comment:
added p2p_order_props.order_id in reactions as notifications count would not be updated when the user has completed rating an order and then navigates out of the order-details page. Even though the order_id has been reset to null when unmounting order_details, the p2p_order_props.order_id in showCompletedOrderNotification function doesn't recognise that p2p_order_props has been updated and won't refresh the notifications. Hence I needed to check it here and refresh the notifications here.